LocaleEmulator/Hooks/Gdi32Hook.cpp

```diff
@@ -10,7 +79,7 @@ HFONT GetFontFromFont(PLeGlobalData GlobalData, HFONT Font)
         return nullptr;
 
     LogFont.lfCharSet = GlobalData->GetLeb()->DefaultCharset;
-    Font = CreateFontIndirectW(&LogFont);
+    Font = LeCreateFontIndirectW(&LogFont);
```

Comment:
I think this custom implementation might be not needed.

The change from MS is a font cache inside `CreateFontIndirectW`, which you can find its address in any crash dump. The crash we now see is a null pointer exception (0x00 + 8), because we call `CreateFontIndirectW` too early before the cache is initialised in `GdiDllInitialize`.

Could you try modifying `LeGetStockObject` in this file to offload requests other than `*_FONT`, i.e.,
```
LeGetStockObject(*_FONT): call `GetFontFromFont`

others: offload to `GlobalData->GetStockObject(Object)`
```
This method does fix the crash from my one experiment, because all legit `*_FONT` calls happens after `GdiDllInitialize` being called by ntdll.

Reply:
Cool. Updated.